### DISCUSSION TITLE
Page does not render if a codeblock has a curly bracket

### DISCUSSION THREAD

**Discussion by mariyabytes** [requarks/wiki#3986] [Error / Bug Report]
In case the markdown of a page has a double curly bracket {{}}, even in a codeblock, the page doesn't render at all. It doesn't error out, but it shows up blank.

#### To reproduce 
Sample markdown / page source: 
https://paste.rs/29o

I've included a pastebin link as github renders the markdown automatically.

#### Expected behaviour
The page should render but it shows up blank, tried on multiple browsers.

#### Host Info
Wiki.js version :  2.5.170
OS: Linux - Ubuntu (groovy) 20.10 x64
Database engine: postgres 12.6
NodeJS : 14.15.3

**Comment by skledarg** [comment]
Your pastebin renders as expected for me on 2.5.201 / postgres 11.12 / NodeJS 14.16.0, but I have a few additional markdown parsers disabled in my rendering pipeline. The ones I have enabled are Core, Task Lists, Emoji, Expand Tabs, Image Size, Subscript/Superscript.

Maybe try disabling some optional markdown parsers one by one and check if you can narrow it down do a specific parser?

EDIT: I can enable all additional markdown modules and the markdown still renders as expected for me.

**Reply by mariyabytes** [reply]
I'll try upgrading to 2.5.201, it might be fixed on a later version. Thank you !
